src/components/Layout/index.jsx

```diff
@@ -53,12 +53,19 @@ export default Layout;
 
 
 Layout.propTypes = {
+  /** Nested structure that needs this wrapper **/
   children: t.node.isRequired,
+  /** The title that will be displayed in the header. **/
```

Comment:
'The title that will be displayed in the page header-bar.'

Reply:
Updated